common/storybook/.storybook/public/mockServiceWorker.js

```diff
@@ -2,13 +2,13 @@
 /* tslint:disable */
 
 /**
- * Mock Service Worker (0.47.3).
+ * Mock Service Worker (0.49.0).
```

Comment:
we're supposed to run `npx msw@0.49.0 init common/storybook/.storybook/public` with whatever version we updated to when we update

but we didn't cos it's silly that it isn't run by npm install